src/docs/transaction-clustering/index.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Sanitizing URL Transactions # TODO: decide on name.
+---
+
+Sentry attempts to scrub high-cardinality identifiers from URL transactions
+to give the user more meaningful transaction groups.
+
+In terms of user experience, this feature plays a similar role as [Issue Grouping](/grouping/).
+In terms of technical implementation, it is similar to [Data Scrubbing](/pii/).
+
+## The Problem
+
+In our [Performance](https://docs.sentry.io/product/performance/) product, transactions are grouped by their name
+(the [`event.transaction`](/sdk/event-payloads/#optional-attributes) field).
+This works well as long as the cardinality of distinct transaction names that the SDK sends is low, for example
+by using the [route of a web framework](https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/) as transaction name.
+
+However, in some cases the SDK does not have enough information to pick a meaningful group
+like the name of a view or a route of a web transaction,
+and it has to fall back to the raw URL (or rather,
+[its path component](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax)).
+
+First and foremost, this makes it harder for the user to extract insights from [Performance
+metrics](https://docs.sentry.io/product/performance/metrics/), because instead of presenting averages, percentiles and distributions of groups of transactions that logically
+belong together, we end up with a bunch of one-off transaction groups.
+
+```
+/account/settings/nf215j634705291cvcm725757m53451365023487cv562n905n2634590xn26340/
+/issues/1234567890
+/saml/acs/my-organization/
+/settings/e1fb593b4c3a4d388e2f6ea8c92c3abc/
+```
+
+Secondly, ingesting a high number of one-off transaction names puts a cost on our infrastructure,
+because with [metrics](https://getsentry.github.io/relay/relay_metrics/index.html), the cost of storage is determined by _cardinality_ (the number of distinct groups), not volume (the number of events ingested).
+
+### << unparameterized >>
+
+An intermediate solution to the cost problem was to drop the transaction tag from Performance metrics
+when the transaction name is a raw URL. This effectively groups _all_ URL transactions into one big group,
+which also creates a bad user experience:
+
+![Project with only unparameterized transactions](./unparameterized.png)
+
+## The Solution
+
+### Pattern-based Identifier Scrubbing
+
+In a first step, Relay strips common identifiers such as UUIDs, integer IDs and hashes
+from URL transactions ([code](https://github.com/getsentry/relay/blob/e543401086c556b769b67ccb96fbfaca411696c8/relay-general/src/store/transactions/processor.rs#L273-L277)).
+For example,
```

Comment:
To be consistent with the `:` above, although I'm not sure what option is grammatically correct.
```suggestion
For example:
```